pages/people.jsx

```diff
@@ -7,6 +7,7 @@ import HowToBecome from 'components/people/HowToBecome/HowToBecome';
 import tugData from '../data/tug_data.js';
```

Comment:
Also, I reckon we should use PascalCase filenames for React components.
Reference: https://github.com/airbnb/javascript/tree/master/react#naming